python/lib/sift_py/rule/service.py

```diff
@@ -159,6 +159,60 @@ def load_rules_from_yaml(
 
         return rule_configs
 
+    def create_or_update_rules(self, rule_configs: List[RuleConfig]):
+        """
+        Create or update a list of rules via a list of RuleConfigs.
+        See `sift_py.rule.config.RuleConfig` for more information on configuation parameters for rules.
+        """
+        for config in rule_configs:
+            self.create_or_update_rule(config)
+
+    def attach_asset(self, rule: Union[str, RuleConfig], asset_names: List[str]):
```

Comment:
Might be nice to return a new RuleConfig which contains the newly attached asset in the `asset_names`?

Reply:
Addressed on [a5f8b29](https://github.com/sift-stack/sift/pull/154/commits/a5f8b2941221df3c0d57b372ad1bb0ca481bed91)! 
One question for you though that came up as I implemented this: https://github.com/sift-stack/sift/pull/154/commits/fe2366c4ca4d2bac1459ab3b7fe95b84e5a1047e#r1890750956